pkg/kv/option.go

```diff
@@ -273,3 +274,6 @@ func getLossyDDLReorgSource(txnSource uint64) uint64 {
 func isLossyDDLReorgSourceSet(txnSource uint64) bool {
 	return (txnSource >> lossyDDLReorgSourceShift) != 0
 }
+
+// WithPipelinedMemDBSkipRemoteBuffer create context will skip remote buffer.
+var WithPipelinedMemDBSkipRemoteBuffer = tikv.WithPipelinedMemDBSkipRemoteBuffer
```

Comment:
This is never used?

Reply:
Yes, it's an risky optimization, you can check the earlier discussion in this PR about this.

Reply:
Since it is never used in TiDB, can we remove it, I'm worried if someone misuses this for performance optimization in the future.